### PR TITLE
Update swedish translation

### LINKED ISSUE #2335: Translate App Strings to Swedish
### User Story

As a Swedish language speaker, I want to be able to interact with Status in my native language.

### Description

*Type*: Feature

*Summary*: We need someone fluent (and eloquent) in Swedish to translate all the latest strings in Status from English. We would like to invite our community members to take on these tasks in order
a) To become a bit more acquainted with the Status repos and perhaps wander further into the codebase and set of issues we need help with, earning yourself Open Bounties all the way down this rabbit hole, and
b) To make sure that the product we deliver has been built by the same community that uses it, in a virtuous feedback loop of increasing understanding and value-generation.

#### Expected behavior
The Swedish translation ought to be as up-to-date as the English one.

#### Actual behavior
We have not yet updated it, preferring to leave this open as an opportunity for our community to get involved again.

### Solution

*Summary*: 

- The file [here](https://github.com/status-im/status-react/blob/develop/src/status_im/translations/sv.cljs) needs to be updated to match the new English one [here](https://github.com/status-im/status-react/blob/develop/src/status_im/translations/en.cljs).
- How you translate is a matter of your preference. There were significant updates to the English file for the 0.9.12 release, with lots of strings both add and REMOVED. We need you to make sure that the files are exact mirrors of each other, except in different languages.
- It might be easier to simply take the new English file and translate from scratch rather than going through the current translated files line by line (though we leave this up to you).

### Additional Information

* Status version: 0.9.12+

There are bounties available for these translations. Please see the QR codes that will be added below after the DevCon 3 announcement.

### PR DESCRIPTION
**Pull request by pafcu**
Fixes #2335 

### Summary:

[comment]: # (Summarise the problem and how the pull request solves it)
Updates Swedish translation to match the current English one

status: ready

### CODE REVIEW

**Comment by andytudhope** [comment]
I will check this in the morning my time from a coverage perspective. Pinging @oskarth to check the actual content. Thank you for the submission @pafcu - are you part of our https://chat.status.im ? If not, please join in and ping me @cryptowanderer to fall further down the rabbit hole 😉

**Comment by andytudhope** [comment]
Hey @pafcu. I have tested this and it seems like you have covered all the bases. Now we need @oskarth to check the actual Swedish content. If you add `:sv` to the group of languages at line 270 of `i18n.cljs` and run `lein test-cljs` you can satisfy yourself that you translated everything. Please do this when you have a moment and add it to this PR?

**Comment by pafcu** [comment]
I tried running the test, but even without modifications it seems to fail.

Log: [error.txt](https://github.com/status-im/status-react/files/1560508/error.txt)

**Comment by pafcu** [comment]
Should I file a separate issue for the failing tests?

**Comment by andytudhope** [comment]
No need @pafcu - no tests are failing on `develop` (though there are some issues with re-frisk) and no tests fail on your branch when I run them, so I suspect it is something you did wrong 😉 Oskar will review sometime this week, just not very high priority and he is a busy guy. He will ping you here when it is done.

File: src/status_im/translations/sv.cljs [hunk 1]
```diff
@@ -2,38 +2,58 @@
 
 (def translations
   {
-   ;common
+   ;;common
    :members-title                         "Medlemmar"
    :not-implemented                       "!inte implementerad"
    :chat-name                             "Chattnamn"
    :notifications-title                   "Aviseringar och ljud"
```
Comment (oskarth) [inline src/status_im/translations/sv.cljs:9]: Notiser och ljud kanske? Se t. ex https://support.apple.com/sv-se/HT201925

File: src/status_im/translations/sv.cljs [hunk 2]
```diff
@@ -2,38 +2,58 @@
...
+                                           :other "{{count}} medlemmar"
                                            :zero  "inga medlemmar"}
+   :public-group-status                   "Offentlig"
    :active-online                         "Online"
    :active-unknown                        "Okänd"
    :available                             "Tillgänglig"
    :no-messages                           "Inga meddelanden"
    :suggestions-requests                  "Begäranden"
    :suggestions-commands                  "Kommandon"
+   :faucet-success                        "Kranbegäran mottagen"
```
Comment (oskarth) [inline src/status_im/translations/sv.cljs]: Kranbegäran känns lite för försvenskat, vad tror du om att kalla det för Faucet? Det är vad heter också. Samma nedan.

File: src/status_im/translations/sv.cljs [hunk 3]
```diff
@@ -55,180 +76,300 @@
...
    :phone-number                          "Telefonnummer"
+   :update-status                         "Uppdatera din status..."
+   :add-a-status                          "Lägg till status..."
+   :status-prompt                         "Lägg till din status. Använd #hashtags för att göra det lättare för andra att hitta dig och tala om det som intresserar dig"
    :add-to-contacts                       "Lägg till i kontakter"
+   :in-contacts                           "I kontakterna"
+   :remove-from-contacts                  "Ta bort från kontakter"
+   :start-conversation                    "Börja en konversation"
+   :send-transaction                      "Sänd en transaktion"
+   :testnet-text                          "Du är på {{testnet}} Testnet. Sänd inte äkta ETH eller SNT till din address"
```
Comment (oskarth) [inline src/status_im/translations/sv.cljs]: Istället för äkta, riktiga kanske? Äkta känns mer som "authentic". Samma nedan isåfall.

File: src/status_im/translations/sv.cljs [hunk 4]
```diff
@@ -55,180 +76,300 @@
...
+
+   ;;sign-up
    :contacts-syncronized                  "Dina kontakter har synkroniserats"
    :confirmation-code                     (str "Tack! Vi har skickat dig ett textmeddelande med en bekräftelse "
                                                "kod. Var god ange den koden för att bekräfta ditt telefonnummer")
    :incorrect-code                        (str "Tyvärr var koden felaktig, var god ange den igen")
    :phew-here-is-your-passphrase          "*Pust* det var svårt, här är din lösenordsfras, *skriv ner det här och förvara det säkert!* Du kommer att behöva det för att återställa ditt konto."
    :here-is-your-passphrase               "Här är din lösenordsfras, *skriv ner det här och förvara det säkert!* Du kommer att behöva det för att återställa ditt konto."
-   :phone-number-required                 "Tryck här för att ange ditt telefonnummer och jag kommer att hitta dina vänner"
+   :here-is-your-signing-phrase           "Här är din underskriftsfras. Du kommer att använde den för att verifiera dina transaktioner. *Skriv ner det här och förvara det säkert!*"
```
Comment (oskarth) [inline src/status_im/translations/sv.cljs]: "Skriv ner den och förvara den säkert!"?

File: src/status_im/translations/sv.cljs [hunk 5]
```diff
@@ -55,180 +76,300 @@
...
+   :enter-address                         "Ge address"
+   :more                                  "mer"
 
-   ;group-settings
+   ;;group-settings
    :remove                                "Ta bort"
    :save                                  "Spara"
+   :delete                                "Ta bort"
    :clear-history                         "Rensa historik"
+   :mute-notifications                    "Tysta anmälningar"
```
Comment (oskarth) [inline src/status_im/translations/sv.cljs]: notiser?

File: src/status_im/translations/sv.cljs [hunk 6]
```diff
@@ -55,180 +76,300 @@
...
+   ;;commands
    :chat-send-eth                         "{{amount}} ETH"
 
-   ;new-group
-
-   ;participants
+   ;;location command
+   :your-current-location                 "Din nuvarande plats"
+   :places-nearby                         "Närliggande platser"
+   :search-results                        "Sök resultat"
```
Comment (oskarth) [inline src/status_im/translations/sv.cljs]: Sökresultat

File: src/status_im/translations/sv.cljs [hunk 7]
```diff
@@ -55,180 +76,300 @@
...
    :You                                   "Du"
 
-   ;new-contact
+   ;;new-contact
    :add-new-contact                       "Lägg till ny kontakt"
-   :scan-qr                               "Skanna QR"
+   :scan-qr                               "Skanna QR kod"
    :name                                  "Namn"
-   :address-explication                   "Kanske borde det finnas lite text som förklarar vad en adress är och var man hittar den"
+   :address-explication                   "Din offentliga nykel används för att generera din address på Ethereum och är en serie siffror och bokstäver. Du kan hitta den lätt i din profil"
```
Comment (oskarth) [inline src/status_im/translations/sv.cljs]: Du kan lätt hitta den i din profil

File: src/status_im/translations/sv.cljs [hunk 8]
```diff
@@ -55,180 +76,300 @@
...
-   :enter-address                         "Ange adress"
-   :send-transaction                      "Skicka transaktion"
-   :delete-contact                        "Radera kontakt"
-   :mute-notifications                    "Stäng av notisljud"
-
-
-   :contact-s                             {:one   "kontakt"
-                                           :other "kontakter"}
-   :next                                  "Nästa"
+   :unsigned-transaction-expired          "Ounderskriven transaktion förföll"
```
Comment (oskarth) [inline src/status_im/translations/sv.cljs]: Ej underskriven

File: src/status_im/translations/sv.cljs [hunk 9]
```diff
@@ -55,180 +76,300 @@
...
+   :got-it                                "OK"
+   :block                                 "Block"
+   :hash                                  "Hash"
+   :gas-limit                             "Gas begränsning"
+   :gas-price                             "Gas pris"
+   :gas-used                              "Gas använt"
+   :cost-fee                              "Kostnad/Avgift"
+   :nonce                                 "Nonce"
+   :confirmations                         "Bekräftelser"
+   :confirmations-helper-text             "Var god och vänta på åtminståne 12 bekräftelser för att försäkra dig att transaktionen behandlas säkert"
```
Comment (oskarth) [inline src/status_im/translations/sv.cljs]: ...på minst...

...försäkra dig om att transaktionen...

File: src/status_im/translations/sv.cljs [hunk 10]
```diff
@@ -55,180 +76,300 @@
...
+   :outgoing                              "Utgående"
+   :pending                               "Pågående"
+   :postponed                             "Uppskjutna"
+
+   ;;webview
+   :web-view-error                        "hoppsan, fel"
 
+   ;;testfairy warning
+   :testfairy-title                       "Varning!"
+   :testfairy-message                     "Du använder en app som installerats från en nattlig version. För att underlätta testning spelar denna version in sessionen om wifi används. All interaktion med appen sparas (som video och loggfiler) och kan användas av utväcklarna för att undersöka möjliga fel. Sparade video/loggfiler innehåller inte lösenord. Inspelningen sker endast om appen installeras från en nattlig version. Inget spelas in om appen installeras från PlayStore eller TestFlight."
```
Comment (oskarth) [inline src/status_im/translations/sv.cljs]: WiFi

utvecklare

File: src/status_im/translations/sv.cljs [hunk 11]
```diff
@@ -55,180 +76,300 @@
...
+   :wallet-send                           "Sänd"
+   :wallet-send-token                     "Sänd {{symbol}}"
+   :wallet-request                        "Begär"
+   :wallet-exchange                       "Växla"
+   :wallet-assets                         "Tillgångar"
+   :wallet-add-asset                      "Lägg till tillgång"
+   :wallet-total-value                    "Totalt värde"
+   :wallet-settings                       "Plånboksinställningar"
+   :wallet-manage-assets                  "Hantera tillgångar"
+   :signing-phrase-description            "Skriv under transaktionen genom att ge ditt lösenord. Kontrollera att orden ovan motsvarar din hemliga underskriftsfras"
```
Comment (oskarth) [inline src/status_im/translations/sv.cljs]: genom att ange ditt

File: src/status_im/translations/sv.cljs [hunk 12]
```diff
@@ -55,180 +76,300 @@
...
+   :usd-currency                          "USD"
+   :transactions                          "Transaktioner"
+   :transaction-details                   "Transaktiondetaljer"
+   :transaction-failed                    "Transaktionen misslyckades"
+   :transactions-sign                     "Skriv under"
+   :transactions-sign-all                 "Skriv under alla"
+   :transactions-sign-transaction         "Skriv under transaktionen"
+   :transactions-sign-later               "Skriv under senare"
+   :transactions-delete                   "Ta bort transaktion"
+   :transactions-delete-content           "Transaktionen tas bort från listan 'Ounderskrivna'"
```
Comment (oskarth) [inline src/status_im/translations/sv.cljs]: 'Ej underskrivna'

File: src/status_im/translations/sv.cljs [hunk 13]
```diff
@@ -55,180 +76,300 @@
...
+   :transactions-history                  "Historik"
+   :transactions-unsigned                 "Ounderskrivna"
```
Comment (oskarth) [inline src/status_im/translations/sv.cljs]: Ej underskrivna

File: src/status_im/translations/sv.cljs [hunk 14]
```diff
@@ -55,180 +76,300 @@
...
+   :transactions-history-empty            "Inga transaktioner in din historik ännu"
```
Comment (oskarth) [inline src/status_im/translations/sv.cljs]: i din historik än

File: src/status_im/translations/sv.cljs [hunk 15]
```diff
@@ -55,180 +76,300 @@
...
+   :transactions-unsigned-empty           "Du har inga ounderskrivna transaktioner"
```
Comment (oskarth) [inline src/status_im/translations/sv.cljs]: ej underskrivna

File: src/status_im/translations/sv.cljs [hunk 16]
```diff
@@ -55,180 +76,300 @@
...
+   :transactions-filter-title             "Filtrera historik"
+   :transactions-filter-tokens            "Tokens"
+   :transactions-filter-type              "Typ"
+   :transactions-filter-select-all        "Välj alla"
+   :view-transaction-details              "Visa transaktionsdetaljer"
+   :transaction-description               "Var god och vänta på åtminståne 12 bekräftelser för att försäkra dig att transaktionen behandlas säkert"
```
Comment (oskarth) [inline src/status_im/translations/sv.cljs]: på minst 12

Comment (oskarth) [reply src/status_im/translations/sv.cljs]: "för att försäkra dig om att transaktionen gått igenom" kanske?

File: src/status_im/translations/sv.cljs [hunk 17]
```diff
@@ -55,180 +76,300 @@
...
+   :transaction-sent                      "Transaktionen sändes"
```
Comment (oskarth) [inline src/status_im/translations/sv.cljs]: skickad

File: src/status_im/translations/sv.cljs [hunk 18]
```diff
@@ -55,180 +76,300 @@
...
+   :transaction-moved-text                "Transaktionen kvarblir på listan 'Ounderskrivna' de nästa 5 minuterna"
```
Comment (oskarth) [inline src/status_im/translations/sv.cljs]: Transaktioner står kvar på listan 'Ej underskrivna' i 5 minuter

File: src/status_im/translations/sv.cljs [hunk 19]
```diff
@@ -55,180 +76,300 @@
...
+   :transaction-moved-title               "Transaktionen flyttades"
+   :sign-later-title                      "Skriv under transaktionen senare?"
+   :sign-later-text                       "Kontrollera transaktionshistoriken för att skriva under denna transaktion"
+   :not-applicable                        "Inte tillämplig på ounderskrivna transaktioner"
```
Comment (oskarth) [inline src/status_im/translations/sv.cljs]: Ej applicerbar på ej underskrivna transaktioner

File: src/status_im/translations/sv.cljs [hunk 20]
```diff
@@ -55,180 +76,300 @@
...
+
+   ;; Wallet Send
+   :wallet-choose-recipient               "Välj mottagare"
+   :wallet-choose-from-contacts           "Välj från kontakter"
+   :wallet-address-from-clipboard         "Använd address i urklipp"
+   :wallet-invalid-address                "Ogiltig address: \n {{data}}"
+   :wallet-invalid-chain-id               "Felaktigt nätverk: \n {{data}}"
+   :wallet-browse-photos                  "Bläddra foton"
+   :validation-amount-invalid-number      "Summan är inte ett giltigt tal"
```
Comment (oskarth) [inline src/status_im/translations/sv.cljs]: belopp?

File: src/status_im/translations/sv.cljs [hunk 21]
```diff
@@ -55,180 +76,300 @@
...
+   :validation-amount-is-too-precise      "Summan är för exakt. Den minsta summan du kan skicka är 1 Wei (1x10^-18 ETH)"
```
Comment (oskarth) [inline src/status_im/translations/sv.cljs]: för precis

**Review by oskarth — CHANGES_REQUESTED** [review]
Tack för ditt bidrag! Mesta ser bra ut, gav lite kommentarer.

File: src/status_im/translations/sv.cljs (hunk 1 above)
Comment (pafcu) [reply src/status_im/translations/sv.cljs:9]: Men det heter Aviseringar på t.ex. Facebook (https://sv-se.facebook.com/help/327994277286267) och Android (https://support.google.com/nexus/answer/6111294?hl=sv)

File: src/status_im/translations/sv.cljs (hunk 5 above)
Comment (pafcu) [reply src/status_im/translations/sv.cljs]: Eller aviseringar. Hursomhelst borde det säkert vara samma som ovan, jo.

File: src/status_im/translations/sv.cljs (hunk 19 above)
Comment (pafcu) [reply src/status_im/translations/sv.cljs]: Är du säker gällande "applicerbar"? SAOL känner inte applicerbar (som låter som en anglism), men däremot tillämplig.

**Comment by pafcu** [comment]
@oskarth Fixed, but feel free to request more changes.

**Comment by andytudhope** [comment]
Cool, am sure Oskar will review again after Christmas. Please can you add the key `:sv` to the `supported-locales` in `i18n.cljs` so that we can be sure we support Swedish fully and that we have full coverage. The def at line 270 should look like this:
```
(def supported-locales (spec/assert ::locales #{:fr
                                                :zh-hans-cn
                                                :zh-hans-sg
                                                :sr-RS_#Cyrl
                                                :zh-hans-tw
                                                :en
                                                :zh
                                                :de
                                                :zh-hans
                                                :lt
                                                :zh-hans-mo
                                                :zh-hans-hk
                                                :sr-RS_#Latn
                                                :sr
                                                :sv}))
```

If you run `lein test-cljs` after that, you will see that all tests pass and that you have indeed covered all the required keys.

**Comment by pafcu** [comment]
@andytudhope I now added `:sv` but as I stated earlier, I can't actually test it, since `lein test-cljs` fails for me in any case.

**Comment by andytudhope** [comment]
Yeah, but that's something wrong in your environment worth looking into 😉 It'll be great if you can prove to yourself that the test are passing! Thanks for adding the key in any case 👍 
Are you sure that you did a successful `npm install` after cloning `status-react`? This part of your error message:
```
Caused by: clojure.lang.ExceptionInfo: node_modules/web3/dist/web3.min.js (No such file or directory) at line 44 src/status_im/utils/js_resources.cljs {:file "src/status_im/utils/js_resources.cljs", :line 44, :column 16, :tag :cljs/analysis-error}
```
suggests that you did not.

**Comment by pafcu** [comment]
@andytudhope Thanks. I (wrongly) assumed that `lein` takes care of all dependencies, but apparently there is another, parallel dependency manager. In any case `npm install` fixed the problem and the test finishes without errors.

File: src/status_im/translations/sv.cljs (hunk 1 above)
Comment (oskarth) [reply src/status_im/translations/sv.cljs:9]: Aha intressant, kan köra på Aviseringar då

File: src/status_im/translations/sv.cljs (hunk 5 above)
Comment (oskarth) [reply src/status_im/translations/sv.cljs]: Aviseringar funkar

**Review by oskarth — APPROVED** [review]

File: src/status_im/translations/sv.cljs (hunk 19 above)
Comment (oskarth) [reply src/status_im/translations/sv.cljs]: http://www.saob.se/artikel/?unik=A_1959-0100.RadQ&pz=5 men Otillämplig funkar också

**Comment by oskarth** [comment]
Tack @pafcu!

**Pull request merged**